optaplanner-examples/src/main/resources/org/optaplanner/examples/conferencescheduling/solver/conferenceSchedulingScoreRules.drl

```diff
@@ -358,4 +357,25 @@ rule "Same day talks"
     then
         scoreHolder.addSoftConstraintMatch(kcontext, - $weight * ($rightTalk.overlappingThemeTrackCount($leftTalk)
             + $rightTalk.overlappingContentCount($leftTalk)));
+end
+
+rule "Popular talks"
```

Comment:
There's 2 ways to formalize the constraint that Stephan was talking about:

- For 2 talks in the same timeslot, the more popular talk needs to be in the bigger room.
- For 2 talks, the more popular talk needs to be in the bigger room. That's what currently is implemented and is probably best. Once we do non-disruptive re-planning, the other hard constrainst will turn that into the first form, but during the initial schedule this ways is indeed better.

I 'd like the name of the constraint to reflect our choice.

Reply:
@ge0ffrey what other hard constraint you're referring to?